platform/contracts/admin/src/contracts/protocol/mod.rs

```diff
@@ -21,6 +21,57 @@ pub struct ProtocolTemplate<T> {
 #[derive(Debug, Clone, Eq, PartialEq, Serialize, Deserialize, JsonSchema)]
 #[serde(rename_all = "snake_case", deny_unknown_fields)]
 pub struct Protocol {
-    pub network: String,
+    pub network: Network,
+    pub dex: Dex,
     pub contracts: ProtocolTemplate<Addr>,
 }
+
+#[derive(Debug, Clone, Eq, PartialEq, Serialize, Deserialize, JsonSchema)]
+#[serde(
+    rename_all = "PascalCase",
+    rename_all_fields = "snake_case",
```

Comment:
as per the [doc](https://serde.rs/container-attrs.html) these two do the same when applied on enums, as it is in our case ..... and you have specified two different styles which would confuse the reader

Reply:
I have a test covering it. It absolutely does NOT do the same thing when they are combined. I've applied them both in order to ensure consistency of the casing of the fields themselves.

Reply:
my bad reading through the doc